presto-native-execution/scripts/release-centos-dockerfile/Dockerfile

```diff
@@ -0,0 +1,162 @@
+# prestissimo
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#============================================================
+# /////////////////////////////////
+#    BUILD TASK DEFINITION
+# /////////////////////////////////
+#============================================================
+ARG BASE_IMAGE=quay.io/centos/centos:stream8
+FROM ${BASE_IMAGE} AS build-stage
+
+ARG CPU_TARGET=avx
+ARG PRESTODB_REPOSITORY=https://github.com/prestodb/presto
+ARG PRESTODB_CHECKOUT=origin/master
+
+USER root:root
+
+ENV CPU_TARGET=${CPU_TARGET}
+ENV TZ=Europe/Warsaw
```

Comment:
can we get this from the env or say use timedatectl ?

Reply:
IT is no longer required. Removed.